fetchai/ledger/contract.py

```diff
@@ -139,7 +133,8 @@ def query(self, api: ContractsApiLike, name: str, **kwargs):
 
         return response['result']
 
-    def action(self, api: ContractsApiLike, name: str, fee: int, *args, signers: Optional[List[Entity]] = None):
+    def action(self, api: ContractsApiLike, name: str, fee: int, signers: List[Entity], *args,
```

Comment:
The `signers` parameter have been relocated from behind in to front of variadic `*args` parameter. The reason for this is because `signers` is mandatory parameter which will MUST be provided in anyway in all cases (there is **no** scenario where it can be possibly omitted).